src/groups/mqb/mqba/mqba_domainmanager.cpp

```diff
@@ -663,27 +688,11 @@ int DomainManager::processCommand(mqbcmd::DomainsResult*        result,
 
         DomainSp domainSp;
 
-        if (0 != locateDomain(&domainSp, name)) {
-            BALL_LOG_WARN << "Domain '" << name << "' is not opened,"
-                          << " trying to initialize from configuration";
-
-            bslmt::Latch latch(1);
-            createDomain(
-                name,
-                bdlf::BindUtil::bind(&onDomain,
-                                     bdlf::PlaceHolders::_1,  // status
-                                     bdlf::PlaceHolders::_2,  // domain*
-                                     &latch));
-            // To return a result from command execution, we need to
-            // synchronize with the domain creation attempt
-            latch.wait();
-
-            if (0 != locateDomain(&domainSp, name)) {
-                mwcu::MemOutStream os;
-                os << "Domain '" << name << "' doesn't exist";
-                result->makeError().message() = os.str();
-                return -1;  // RETURN
-            }
```

Comment:
Hi @lukedigiovanna this refactor with extracting code to `locateOrCreateDomain` might be done in a separate branch. If you do so, we will be able to merge that small branch very fast, and you will be able to trigger GH CI automatically without anyone's approval.
Do you want to try it?

Reply:
Sure! To clarify - I would just make this change and make a PR for it? Is there a reason to do this as opposed to just leaving it in this PR? Is it because this change is not exactly related to the objective of this PR?

Reply:
>I would just make this change and make a PR for it?
Yes, you need to copy 2 files with changes from this PR: `src/groups/mqb/mqba/mqba_domainmanager.cpp` and `src/groups/mqb/mqba/mqba_domainmanager.h`, and move these files to another branch. Then open a second PR with another branch.

> Is there a reason to do this as opposed to just leaving it in this PR?
The main reason why I proposed this - to unblock you from using CI. Currently, if you want to check CI messages, someone else from the team has to press a button to approve CI run. If you contribute something to the repo, approval would not be needed, so CI will start automatically if you push anything to the branch.
Also, the reason could be to make this PR smaller. Typically it's better to split huge PRs into smaller chunks, if it's possible. It will make the review process easier, since a reviewer will need to keep in mind less things at the same time while looking at the branch. However, it's not strictly necessary.

>Is it because this change is not exactly related to the objective of this PR?
We don't have a strict requirement to keep not relevant changes in separate PRs. Moreover, the good pattern is to fix a few things around your code when you make changes, if you see something you can easily fix. Typos, wrong formatting, outdated comments, warnings etc, it's too much to create a separate branch for any of these, but adding a few of this changes in your feature branch might be okay to improve overall code quality.